File: cpp_code/practical/cache/LocalCache.cpp
/**
* 抖码算法，让算法学习变的简单有趣
* @作者 : 老汤
*/

#include <unordered_map>

using namespace std;

class LocalCache {
private:
    unordered_map<int, int> cache;

public:
    int get(int key) {
        return cache[key];
    }

    int put(int key, int value) {
        return cache[key] = value;
    }
};

File: python_code/practical/cache/LocalCache.py
# 抖码算法，让算法学习变得简单有趣
# 作者：老汤


class LocalCache:
    def __init__(self):
        self.cache = dict()

    def get(self, key):
        return self.cache[key]

    def put(self, key, value):
        self.cache[key] = value